src/lib/utils/toc.ts

```diff
@@ -128,6 +129,6 @@ const addHeadingsAsItems = (headings: Array<string>, h = 2): Array<ToCItem> => {
 export const generateTableOfContents = (content: string): Array<ToCItem> => {
   const contentWithoutComments = removeMarkdownComments(content)
   const lines = contentWithoutComments.split("\n")
-  const headings = lines.filter((line) => line.startsWith("#"))
+  const headings = lines.filter((line) => line.startsWith("##"))
```

Comment:
A comment here would be helpful for future reference and context. Why double `#`?

From your PR comments, we could add: `"each page should only have one h1, and it is not shown in the ToC"`

Reply:
Sure! Will add a comment